statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/ResourceHandler.java

```diff
@@ -85,7 +85,7 @@ protected void verifyStateMachineArnIsPresent(String resourceArn) throws AmazonS
         }
     }
 
-    private AmazonServiceException getStateMachineDoesNotExistException() {
+    protected AmazonServiceException getStateMachineDoesNotExistException() {
```

Comment:
question: what motivated this change?

Reply:
This change was included as part of https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-stepfunctions/pull/21. I have merged the changes from `main` into the branch, and these are no longer included as part of the file diff.